src/Storefront/Resources/app/storefront/src/plugin/listing/filter-range.plugin.js

```diff
@@ -146,7 +148,8 @@ export default class FilterRangePlugin extends FilterBasePlugin {
         if (this._inputMin.value.length || this._inputMax.value.length) {
             if (this._inputMin.value.length) {
                 labels.push({
-                    label: `${this.options.snippets.filterRangeActiveMinLabel} ${this._inputMin.value} ${this.options.currencySymbol}`,
+                    /** @deprecated tag:v6.5.0 - currencySymbol will be removed */
+                    label: `${this.options.snippets.filterRangeActiveMinLabel} ${this._inputMin.value} ${this.options.currencySymbol ?? this.options.unit}`,
```

Comment:
There is a second if-block in line 154. Please add these changes also there.
And for better readability:
```suggestion
                    /** @deprecated tag:v6.5.0 - `currencySymbol` will be removed */
                    label: `${this.options.snippets.filterRangeActiveMinLabel} ${this._inputMin.value} ${this.options.currencySymbol ?? this.options.unit}`,
```

Reply:
Ah, I overlooked the second place. Sorry!